zetaclient/evm/evm_signer.go

```diff
@@ -297,16 +282,16 @@ func (signer *Signer) SignCommandTx(
 		if err != nil {
 			return nil, err
 		}
-		tx, _, _, err := signer.Sign(data, to, gasLimit, gasPrice, outboundParams.OutboundTxTssNonce, height)
+		tx, _, _, err := signer.Sign(data, txData.to, txData.gasLimit, txData.gasPrice, outboundParams.OutboundTxTssNonce, txData.height)
 		if err != nil {
 			return nil, fmt.Errorf("sign error: %w", err)
 		}
 		return tx, nil
 	}
-	if cmd == common.CmdMigrateTssFunds {
-		tx := ethtypes.NewTransaction(outboundParams.OutboundTxTssNonce, to, outboundParams.Amount.BigInt(), outboundParams.OutboundTxGasLimit, gasPrice, nil)
+	if txData.cmd == common.CmdMigrateTssFunds {
+		tx := ethtypes.NewTransaction(outboundParams.OutboundTxTssNonce, txData.to, outboundParams.Amount.BigInt(), outboundParams.OutboundTxGasLimit, txData.gasPrice, nil)
 		hashBytes := signer.ethSigner.Hash(tx).Bytes()
```

Comment:
It is a bit strange, the Migrate TSS funds is creating and signing the tx directly in the logic instead of calling the sign function. I think that the Migrate command can also just call the higher level sign function instead of signing directly with the tssSigner.